shared/js/background/redirect.es6.js

```diff
@@ -120,6 +121,14 @@ function handleRequest (requestData) {
         }
 
         if (tracker) {
+            // temp allowlisted trackers to fix site breakage
+            const allowListed = trackerAllowlist(thisTab.site.url, requestData.url)
```

Comment:
I think there's also `thisTab.site.domain` which is already eTLD+1 which would save you needing to parse inside the `trackerAllowlist` function.

Reply:
`site.domain` can include the subdomain. I wanted allowlist matching to be on the root domain only. This way we don't need to worry about allowlisting every subdomain of a site that might be breaking.